src/main/kotlin/org/jmailen/gradle/kotlinter/tasks/LintTask.kt

```diff
@@ -26,6 +27,10 @@ open class LintTask : SourceTask() {
     @PathSensitive(PathSensitivity.RELATIVE)
     lateinit var reports: Map<String, File>
 
+    @InputFiles
+    @PathSensitive(PathSensitivity.RELATIVE)
+    override fun getSource() = super.getSource()
```

Comment:
The desirable path sensitivity does not get inherited from `SourceTask` so it looks like we need to override just to annotate it.